### ISSUE TITLE
unable to delete empty buffers

### ISSUE THREAD

**Issue by spiderbit** [ch11ng/exwm#82]
all few days everything freeses while I am in a conkeror exwm buffer.

thats bad enough but I dont see a way to reproduce it, so for now I have a different easier to resolv problem at least I guess that its easier to solv:

To get emacs back to a state where it reacts to input, I have to go to tty and kill "xulrunner", then the the buffer still exists but its empty.

When I try to kill the buffer I get following error/warning:
`Sitename - Conkeror' is not responding. Would you like to kill it? (y or n) y
if: This method is intended for requests with replies

then the Buffer is not gone, if I try to open then a new conkeror buffer/instance it just doesnt do it.
So I have to quit emacs restart it with startx to be able to start new conkeror windows.

so 1. I want to be able to get rid of this empty buffers
2. I want to be able to start new conkeror exwm buffers.

I assume that the 2nd does not work because of the 1. but maybe I am wrong.

```elisp
(global-set-key (kbd "<menu> e")
                      (lambda ()
                        (interactive)
                        (start-process-shell-command "conkeror" nil "conkeror")))
```

thats how I start conkeror if thats importent

btw I belive that hangs are Related to: https://github.com/ch11ng/exwm/issues/78
I think your fix just removes this error messagee but not the cause of the hanging

But I am not shure maybe that was a differnt bug.

**Comment by ch11ng** [comment]
I do encounter the problem sometimes. Usually the buffer should get killed no matter how the application was terminated. It's most probably because EXWM had some difficulty in destroying the X windows of this application, and the application in turn was not terminated successfully. As a result, a new instance of this application would not be able to start (perhaps because some resources were still in hold by the older instance, or simply because EXWM was not functioning normally). It'd be helpful if we can find a way to reproduce this issue.

**Comment by spiderbit** [comment]
I have some updates, 

1. its not only conkeror I cant start aftr that "freezes" if I then try to start a urxvt window it does not work too. its more like emacs does not allow the start of any exwm buffers anymore.

2. get following messags when it freezes:
error in process filter: if: This method is intended for requests with replies
error in process filter: This method is intended for requests with replies

**Comment by ch11ng** [comment]
So it's because EXWM does not function normally after that fatal error.  But I still don't see the cause of that error.

**Comment by ch11ng** [comment]
I think it has been fixed in b2ee10205177361a5f14dd11c4206b5ffd1cb98c and ch11ng/xelb@35a055dfc825fec0101e2730d655aaecc930f193.
